AutoCollection/CorrelationContextManager.ts

```diff
@@ -1,4 +1,5 @@
-/// <reference path="..\node_modules\zone.js\dist\zone.js.d.ts" />
+/// <reference path="../node_modules/zone.js/dist/zone.js.d.ts" />
```

Comment:
I think this is required not because of the version of zone but the strange way I require it without importing so that it is not loaded on incompatible systems. Maybe a tsconfig can change this?